contracts/base/Dispatcher.sol

```diff
@@ -308,8 +308,12 @@ abstract contract Dispatcher is Payments, V2SwapRouter, V3SwapRouter, Callbacks
                             amount := calldataload(add(inputs.offset, 0x60))
                         }
                         Payments.sweepERC1155(token, recipient.map(), id, amount);
+                    } else if (command == Commands.SEAPORT_V2) {
+                        (uint256 value,) = abi.decode(inputs, (uint256, bytes));
+                        bytes calldata data = inputs.toBytes(1);
+                        (success, output) = SEAPORT_V2.call{value: value}(data);
```

Comment:
I think we should have a very clear comment here and in the readme about the fact that this contract is _not_ secure if the `offerer` of the order is a contract, and a very quick high level sentence explaining the attack, and a link to their doc that talks about it

Reply:
```
                /// @dev Seaport 1.2 allows for orders to be created by contracts. 
                ///     These orders pass control to the contract offerers during fufillment,
                ///         allowing them to perform any number of destructive actions as a holder of the NFT. 
                ///     Integrators should be aware that in some scenarios: e.g. purchasing an NFT that allows the holder
                ///         to claim another NFT, the contract offerer can "steal" the claim during order fufillment. 
                ///     This is also outlined in the Seaport documentation: https://github.com/ProjectOpenSea/seaport/blob/main/docs/SeaportDocumentation.md
```